polymer-redux.js

```diff
@@ -19,11 +19,12 @@
                     var splices = [];
                     var value, previous;
 
-                    // statePath a path or function
-                    if (typeof property.path == 'function') {
-                        value = property.path.call(element, state);
+                    // statePath, a path or function.
+                    var path = property.path;
+                    if (typeof path == 'function') {
+                        value = path.call(element, state);
                     } else {
-                        value = Polymer.Base.get(property.path, state);
+                        value = Polymer.Base.get(path, state);
```

Comment:
I first wanted to replace `value = Polymer.Base.get(path, state);` with reduce, but the test failed. Post it here in case you need it some time.

```javascript
value = path.split(".").reduce(function (p, c) {
                            // If previous path was undefined, this sub path is also undefined.
                            return p && p[c];
                        }, state);
```
ES6
```javascript
value = path.split(".").reduce((p, c) => p && p[c], state);
```